deploy/kubernetes/overlays/prow-gke-release-staging-rc/enable_provisioner_extra_metadata.yaml

```diff
@@ -0,0 +1,4 @@
+# for external-provisioner
+- op: add
+  path: /spec/template/spec/containers/0/args/-
+  value: "--extra-create-metadata"
```

Comment:
It's not enough to just enable the flag. We actually need to plumb the information that's passed in to PD creation.

Reply:
Right. Plumbing the information is a longer term task, and for the time being I wanted to first enable this flag and let it soak.